code/modules/food_and_drinks/food/snacks.dm

```diff
@@ -2478,3 +2478,11 @@
 	icon_state = "tatortot"
 	list_reagents = list("nutriment" = 4)
 	filling_color = "FFD700"
+
+/obj/item/weapon/reagent_containers/food/snacks/cucumberslice
+	name = "Cucumber Slice"
+	desc = "A slice of cucumber"
+	icon_state = "cucumberslice"
+	list_reagents = list("nutriment" = 1)
```

Comment:
You mean like I added back in 2015?
https://github.com/ParadiseSS13/Paradise/pull/1112
Yea, that'd sure be nice, huh?